diff --git a/src/benchmark.h b/src/benchmark.h
index 7b66751496..4aea206148 100644
--- a/src/benchmark.h
+++ b/src/benchmark.h
@@ -34,14 +34,6 @@
 #if defined(HAVE_CLOCK_GETTIME) && defined(CLOCK_PROCESS_CPUTIME_ID)
 #undef gettime
 #define gettime(x) clock_gettime(CLOCK_PROCESS_CPUTIME_ID, x)
-#else
-inline static void gettime(struct timespec *ts)
-{
-	struct timeval tv;
-	gettimeofday(&tv, NULL);
-	ts->tv_sec = tv.tv_sec;
-	ts->tv_nsec = tv.tv_usec * 1000;
-}
 #endif
 
 typedef void (*sighandler_t)(int);
