diff --git a/Library/Homebrew/extend/ENV/super.rb b/Library/Homebrew/extend/ENV/super.rb
index 89f7ecfa5e..b3540c4b6a 100644
--- a/Library/Homebrew/extend/ENV/super.rb
+++ b/Library/Homebrew/extend/ENV/super.rb
@@ -60,6 +60,7 @@ module Superenv
     super
     send(compiler)

+    self["MACOSX_DEPLOYMENT_TARGET"] = "10.15"
     self["HOMEBREW_ENV"] = "super"
     self["MAKEFLAGS"] ||= "-j#{determine_make_jobs}"
     self["RUSTFLAGS"] = Hardware.rustflags_target_cpu(effective_arch)
diff --git a/Library/Homebrew/extend/os/mac/hardware.rb b/Library/Homebrew/extend/os/mac/hardware.rb
index d984567c69..2d4d68fa26 100644
--- a/Library/Homebrew/extend/os/mac/hardware.rb
+++ b/Library/Homebrew/extend/os/mac/hardware.rb
@@ -4,6 +4,7 @@
 module Hardware
   sig { params(version: T.nilable(Version)).returns(Symbol) }
   def self.oldest_cpu(version = nil)
+    return :core2
     version = if version
       MacOSVersion.new(version.to_s)
     else
